Server/src/main/java/utilities/Constants.java

```diff
@@ -8,6 +8,9 @@
     public static final String WARNING_THRESHOLD = "Warning";
     public static final String FAILED_THRESHOLD = "Failed";
 
+    public static final int STATUS_LIVE = 1;
+    public static final int STATUS_EVALUATION = 2;
```

Comment:
I am guessing these constants are gonna be used in the next issues on model parameters to make use of these?

Reply:
yep